[DOCS-12980] Add agentConfiguration parameter instructions (#33582)
DataDog/documentation: content/en/integrations/guide/azure-advanced-configuration.md

**Note**: If `agentConfiguration` and `api_key` are specified at the same time, the API key found in the `agentConfiguration` takes precedence. Also note that if an API key is set on the target machine, it's not possible to change it with `Set-AzVMExtension`.

### Specify a configuration URI

This example shows how to specify a configuration for the Datadog Agent to use.
The Datadog Agent configuration URI must be an Azure blob storage URI.
The Datadog Windows Agent Azure Extension checks that the `agentConfiguration` URI comes from the `.blob.core.windows.net` domain.
The Datataog Agent configuration should be created from the `%PROGRAMDATA%\Datadog` folder (see the [sample `config_template.yaml` file][101] for all available configuration options).

<div class="alert alert-info">

> You can reuse the configuration of an existing Agent:

Reviewer: ```suggestion
To reuse the configuration of an existing Agent:
```

**Note**: If `agentConfiguration` and `api_key` are specified at the same time, the API key found in the `agentConfiguration` takes precedence. If an API key is set on the target machine, it's not possible to change it with the `api_key` setting.

### Specify a configuration URI

This example shows how to specify a configuration for the Datadog Agent to use.
- The Datadog Agent configuration URI must be an Azure blob storage URI.
- The Datadog Linux Agent Azure Extension checks that the `agentConfiguration` URI comes from the `.blob.core.windows.net` domain.
- The Datataog Agent configuration should be created from the `/etc/datadog-agent/` folder (see the [sample `config_template.yaml` file][201] for all available configuration options).

<div class="alert alert-info">

> You can reuse the configuration of an existing Agent by saving its <code>/etc/datadog-agent</code> folder as a ZIP file:

Reviewer: ```suggestion
To reuse the configuration of an existing Agent by saving its <code>/etc/datadog-agent</code> folder as a ZIP file:
```
